<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSelectorUi</class>
 <widget class="QDialog" name="LevelSelectorUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Level</string>
  </property>
  <property name="Level" stdset="0">
   <string>LevelSelectorUi</string>
  </property>
  <widget class="QPushButton" name="btn_change_dir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>261</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>229</width>
      <height>259</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="grid">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
